.github/workflows/deploy-dev.yml

```diff
@@ -76,4 +71,28 @@ jobs:
           azure-client-id: ${{ secrets.AZURE_CLIENT_ID }}
           azure-tenant-id: ${{ secrets.AZURE_TENANT_ID }}
           azure-subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-          app-name: reportvision
\ No newline at end of file
+          app-name: reportvision
+
+  deploy-services:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.deploy-env }}
+    needs: [build-publish-ocr, build-frontend, environment-setup]
```

Comment:
deploying both the frontend and ocr-api only after they are both built and the infrastructure is setup with Terraform.